zh.js

```diff
@@ -29,20 +29,20 @@ generic.failed 失败
 generic.success 成功！
 generic.name 用户名
 generic.level 等级
-generic.score 总得分
+generic.score 得分
```

Comment:
This might be confused with line 197: "settings.interface.score Show Score Counter" as this accumulates the total score you earned since you created your account whereas line 197 just show the score of the current match.